WordPress/gutenberg
# Thunkify reusable-blocks store

This PR migrates the reusable-blocks data store to thunks.

I reworked the unit tests to check for outcomes instead of the specific flow we tested for inside of the generator.

**How to test:**
Open the post editor, create a reusable block, reload the page, convert back to static, create another one, delete it, and confirm everything worked as expected.
